alf/trainers/policy_trainer.py

```diff
@@ -521,6 +521,262 @@ def _eval(self):
         self._algorithm.train()
 
 
+class OfflineRLTrainer(Trainer):
+    """Trainer for reinforcement learning. """
+
+    def __init__(self, config: TrainerConfig, ddp_rank: int = -1):
```

Comment:
Looks very similar to RLTrainer. Is it possible to reuse some code?

Reply:
> Looks very similar to RLTrainer. Is it possible to reuse some code?

Yes, that's true. The current ``OfflineRLTrainer`` is indeed modified from ``RLTrainer`` with some of its own control logic embedded. 

More sharing will be better and I will push it more toward this goal.